lib/wallet/cardano-wallet.cabal

```diff
@@ -115,6 +115,7 @@ library
     , filepath
     , fmt
     , foldl
+    , formatting
```

Comment:
```suggestion
```

I would like to limit our dependencies, both for technical and cognitive reasons — we already use `fmt` as formatting library. As far as I can tell, `listF` and `jsonListF` are suitable functions for formatting lists. The function `genericF` also works if you feel like omitting boilerplate.

Reply:
this is sad, formatting is more understandable ...

Reply:
🤷 I'm not too fond of `fmt` either. However, the choice of formatting library was decided before my time, and we need to stick to it until we make a new decision.